languagetool-language-modules/nl/src/main/java/org/languagetool/rules/nl/CompoundAcceptor.java

```diff
@@ -558,14 +558,14 @@ public class CompoundAcceptor {
     }
   }
 
-  private final Tagger tagger;
+  private final DutchTagger DutchTagger;
 
   CompoundAcceptor() {
-    tagger = Languages.getLanguageForShortCode("nl").getTagger();
+    this.DutchTagger = new DutchTagger();
```

Comment:
We prefer to avoid creating new instances. Use the previous version. I will add a default tagger for NL.

Reply:
Done here. https://github.com/languagetool-org/languagetool/commit/1e445e19d9cc2f547e752f685d3d7125d557c3e9